_doc-build-windows/action.yml

```diff
@@ -211,47 +211,95 @@ runs:
 
     # ------------------------------------------------------------------------
 
+    - uses: ansys/actions/_logging@main
+      with:
+        level: "INFO"
+        message: >
+          Determine context.
+
+    - name: "Determine Github environnement variables"
+      shell: powershell
+      run: |
+        if ((Test-Path "pyproject.toml") -and (Get-Content "pyproject.toml" | Select-String -Pattern 'build-backend = "poetry\.core\.masonry\.api"')) {
+          echo "BUILD_BACKEND=$(echo 'poetry')" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+          echo "SPHINX_BUILD_MAKE=$(echo 'poetry run -- doc\make.bat')" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        } else {
+          echo "BUILD_BACKEND=$(echo 'pip')" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+          echo "SPHINX_BUILD_MAKE=$(echo 'doc\make.bat')" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
Why the change of location for Windows? On Linux you were assuming the make commands to be ran at "doc" level. I would expect the same behavior on both actions.

Reply:
The thing is that on Linux we leverage `make` with option `-C` to "change directory before doing anything". We can't do the same with Windows as we are directly calling the `make.bat` file.
If you want to have the same behavior for both actions, I can add a "move" step before building the documentation. Would you prefer that ?

Reply:
In my experience... context matters when running these batch script files. Sometimes they are not properly adapted to be ran from a different directory (which is what you are doing on the Windows version). I'd recommend changing directories, yeah.

Reply:
Fine by me, in that case I'll avoid using `-C` with linux and also move to the `doc` folder. If you have any other feedback, let me now !

Reply:
``make -C`` is effectively changing directory for you. so that's fine. You do not need to do it yourself on Linux. See https://www.man7.org/linux/man-pages/man1/make.1.html

make.bat is a batch script with which you are trying to mimic the Make GNU program. So, that's why the context is important in that case

Reply:
While starting the modifications, I noticed that the steps associated to assets would be impacted by the changes.
Currently, this steps define `EXPECTED_BUILD_DIR=doc/_build` for Linux and `EXPECTED_BUILD_DIR=doc\_build` for Windows. While I do agree with your previous comment, I'd rather perform a manual folder move with `cd doc` in both Windows and Linux. Otherwise we would end up with "different" environment variable which might become confusing later on. @RobPasMue Is that fine with you ?

Reply:
Sure, go ahead

Reply:
After testing I'll keep `make -C` for Linux (previous behavior). I though that it might be confusing with `EXPECTED_BUILD_DIR` but I forgot the fact that every step of the job starts at the root level. Even if in a previous step I did `cd doc` :)